app/docs/notes/Setu-Peppol.md

```diff
@@ -0,0 +1,29 @@
+
+## SETU and PEPPOL
+
+Through agreements between SETU and the Dutch Peppol Authority (NPa), standard SETU XML messages can now be sent using the PEPPOL network infrastructure.
+
+To enable this, specific **Document Type Identifiers** have been officially registered within the PEPPOL network for the various SETU message types. These identifiers are crucial for the **identification and processing** of the messages within the network. They ensure that the receiving system recognizes the type of SETU message and can process its content accordingly. **The image below provides an example overview of the Document Type Identifiers assigned to SETU messages. For the complete, authoritative list, please refer to the official PEPPOL Codelists page mentioned below.**  
+
+<br/>
+
+![](../../static/img/peppol%20document%20types.png)
+
+These identifiers are registered in the receiver's Service Metadata Publisher (SMP), which acts as an address book within PEPPOL. The SMP lists which document types an organization can receive and via which Access Point. When a message is sent, the sending AP queries the central Service Metadata Locator (SML) and then the receiver's SMP to verify that the recipient can handle the specific SETU message type and to determine the correct delivery endpoint (the receiver's AP).
+
+## How to Use SETU via PEPPOL
+
+To send or receive SETU messages via the PEPPOL network:
+
+1. **Connect to a PEPPOL Access Point Provider:** Your organization needs a connection to the PEPPOL network via a certified PEPPOL Access Point (AP) provider. You can find a list of providers operating in the Netherlands via the NPa website. While technically possible, implementing your own AP is significantly more complex.
+2. **Use the Correct Document Type Identifiers:** Ensure that your software and/or your Access Point provider are configured to use the correct, officially registered Document Type Identifiers for the specific SETU messages you intend to exchange. This Document Type Identifier should be included within the PEPPOL envelope when sending the message. The complete, authoritative list of all official PEPPOL Document Type Identifiers (including those for SETU) is maintained on the [official PEPPOL Codelists page](https://docs.peppol.eu/edelivery/codelists/). Refer to the image example above for the SETU-specific identifiers.
+3. **Agree with Trading Partners:** Coordinate with your business partners to confirm that you will be exchanging SETU messages using the PEPPOL network.
```

Comment:
Trading partners is the term used by PEPPOL. Let's use SETU terminology here: `staffing customers and/or staffing suppliers` or use the neutral term `business partners`